test/Ionide.ProjInfo.Tests/Tests.fs

```diff
@@ -106,23 +106,24 @@ module ExpectNotification =
         sprintf "failed %s" name, isFailed
 
     let expectNotifications actual expected =
-        Expect.equal (List.length actual) (List.length expected) (sprintf "notifications: %A" (expected |> List.map fst))
+
+        let getMessages =
+            function
+            | WorkspaceProjectState.Loading (path) -> sprintf "loading %s " path
+            | WorkspaceProjectState.Loaded (po, _, _) -> sprintf "loaded %s" po.ProjectFileName
+            | WorkspaceProjectState.Failed (path, _) -> sprintf "failed %s" path
+
+        Expect.equal (List.length actual) (List.length expected) (sprintf "expected notifications: %A \n actual notifications: - %A" (expected |> List.map fst) (actual |> List.map getMessages))
 
         expected
-        |> List.zip actual
         |> List.iter
-            (fun (n, check) ->
-                let name, f = check
-
-                let minimal_info =
-                    match n with
-                    | WorkspaceProjectState.Loading (path) -> sprintf "loading %s " path
-                    | WorkspaceProjectState.Loaded (po, _, _) -> sprintf "loaded %s" po.ProjectFileName
-                    | WorkspaceProjectState.Failed (path, _) -> sprintf "failed %s" path
+            (fun (name, f) ->
```

Comment:
The new graph solution may order messages slightly different due to it's sorting. I had to make this more flexible with items potentially being out of order.